includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1596,6 +1596,19 @@ public function update_option( $key, $value = '' ) {
 		return parent::update_option( $key, $value );
 	}
 
+	/**
+	 * Updates whether platform checkout is enabled or disabled.
+	 *
+	 * @param bool $is_platform_checkout_enabled Whether platform checkout should be enabled.
+	 */
+	public function update_is_platform_checkout_enabled( $is_platform_checkout_enabled ) {
+		$current_is_platform_checkout_enabled = 'yes' === $this->get_option( 'platform_checkout', 'no' );
+		if ( $is_platform_checkout_enabled !== $current_is_platform_checkout_enabled ) {
+			wc_admin_record_tracks_event( $is_platform_checkout_enabled ? 'platform_checkout_enabled' : 'platform_checkout_disabled' );
```

Comment:
Looks like we're using that prefix for all events in WCPay except for `page_view`. [Here are the ones](https://github.com/Automattic/woocommerce-payments/blob/develop/client/tracks/index.js#L31) recorded on the client-side, for example. So I'm wondering if the event name should be prefixed with `wcpay` so it ends up being `wcadmin_wcpay_platform_checkout_{action}`.

Not sure if this was discussed or we decided not to use it in this case, though. What do you think?

Reply:
Thanks for raising the question — for these events, we've been excluding `wcpay` from the event name so that platform checkout events could be recorded in other places in the future, even though they'll only be in WCPay for now.

Reply:
Makes sense. Thanks for clarifying!